cluster/examples/vm-alpine-datavolume.yaml

```diff
@@ -14,6 +14,7 @@ spec:
       pvc:
         accessModes:
         - ReadWriteOnce
+        dataSource: null
```

Comment:
I understand this new field is not required, right? If so, I would like to get rid of these null fields in the example manifests. Can you please remove it? See https://github.com/kubevirt/kubevirt/blob/master/tools/util/marshaller.go#L46

Reply:
The three null fields should now be gone. I hope to code to remove them is ok!